web/src/components/__tests__/__snapshots__/Header.spec.tsx.snap

```diff
@@ -22,25 +22,25 @@ exports[`Header should match snapshot with largeViewport 1`] = `
         link="/random_route"
```

Comment:
Could you perhaps remove the snapshot and write a new test for this?

Reply:
we can create tickets for this. as i also touched other components i would have to adjust more than just the header text which is in my opinion out of scope for this ticket

Reply:
I just meant the snapshot test, but of course this is never a must do, always a can do. So it is fine :)